### PR TITLE
Clarify named config scope nesting

### PR DESCRIPTION
**Pull request by llewellyn-sl**
-Add clarification and example of named config scope nesting inside a config profile.  -Minor language fixes.

Signed-off-by: Llewellyn vd Berg <113503285+llewellyn-sl@users.noreply.github.com>

### CODE REVIEW

File: docs/config.rst [hunk 1]
```diff
@@ -20,11 +18,11 @@ of priority:
 6. The config file ``$HOME/.nextflow/config``
 7. Values defined within the pipeline script itself (e.g. ``main.nf``)
 
-When more than one of these ways of specifying configurations are used, they are merged, so that the settings in the
-first override the same ones that may appear in the second one, and so on.
+When more than one of these options for specifying configurations are used, they are merged, so that the settings in the
+first override the same settings that appearing in the second, and so on.
```
Comment (llewellyn-sl) [inline docs/config.rst:22]:
```suggestion
first override the same settings appearing in the second, and so on.
```

File: docs/config.rst [hunk 2]
```diff
@@ -35,14 +33,14 @@ A Nextflow configuration file is a simple text file containing a set of properti
   name = value
 
 Please note, string values need to be wrapped in quotation characters while numbers and boolean values (``true``, ``false``) do not.
-Also note that values are typed, meaning for example that, ``1`` is different from ``'1'``, since the first is interpreted
+Also note that values are typed. THis means that, for example, ``1`` is different from ``'1'`` — the first is interpreted
```
Comment (llewellyn-sl) [inline docs/config.rst:36]:
```suggestion
Also note that values are typed. This means that, for example, ``1`` is different from ``'1'`` — the first is interpreted
```

File: docs/config.rst [hunk 3]
```diff
@@ -35,14 +33,14 @@ A Nextflow configuration file is a simple text file containing a set of properti
...
 as the number one, while the latter is interpreted as a string value.
 
 
 Config variables
 ----------------
 
-Configuration properties can be used as variables in the configuration file itself, by using the usual
+Configuration properties can be used as variables in the configuration file, by using the usual
```
Comment (llewellyn-sl) [inline docs/config.rst:43]:
```suggestion
Configuration properties can be used as variables in the configuration file by using the
```

File: docs/config.rst [hunk 4]
```diff
@@ -960,9 +957,25 @@ to the same profile using a common prefix. For example::
 
     }
 
-This configuration defines three different profiles: ``standard``, ``cluster`` and ``cloud`` that set different process
-configuration strategies depending on the target runtime platform. By convention the ``standard`` profile is implicitly used
-when no other profile is specified by the user.
+This configuration defines three different profiles: ``standard``, ``cluster``, and ``cloud``, that each set different process
+configuration strategies depending on the target runtime platform. The ``standard`` profile is used by default when no profile is specified by the user.
```
Comment (llewellyn-sl) [inline docs/config.rst:961]:
```suggestion
configuration strategies depending on the target runtime platform. The ``standard`` profile is used by default when no profile is specified. 
```

**Review by bentsherman — APPROVED** [review]

File: docs/config.rst [hunk 5]
```diff
@@ -960,9 +957,25 @@ to the same profile using a common prefix. For example::
...
+        timeline.enabled = true
+        
+    // This defines an AWS scope within the profile named aws
+        aws {
+          client {
+            maxConnections = 10
+          }
+        }
+      }
+    }
```
Comment (pditommaso) [inline docs/config.rst:978]:
```suggestion
```

Comment (pditommaso) [reply docs/config.rst:978]: Syntax in this example is broken. I've removed it.

**Pull request merged**